improve accelerate reference in docs (#1086)
TimDettmers/bitsandbytes: docs/source/integrations.mdx

# Transformers

With Transformers it's very easy to load any model in 4 or 8-bit, quantizing them on the fly with `bitsandbytes` primitives.

> Please review the [`bitsandbytes` section in the Accelerate docs](https://huggingface.co/docs/transformers/v4.37.2/en/quantization#bitsandbytes).

Reviewer: ```suggestion
Please review the [`bitsandbytes` section in the Transformers docs](https://huggingface.co/docs/transformers/main/en/quantization#bitsandbytes).
```

Author: good catch!

# PEFT
With `PEFT`, you can use QLoRA out of the box with `LoraConfig` and a 4-bit base model.

> Please review the [`bitsandbytes` section in the Accelerate docs](https://huggingface.co/docs/peft/developer_guides/quantization#quantize-a-model).

Reviewer: ```suggestion
Please review the [`bitsandbytes` section in the PEFT docs](https://huggingface.co/docs/peft/developer_guides/quantization#quantize-a-model).
```

Author: ah, I noticed that too, but fixed it in `main` so that the correction is live immediately..

# Accelerate

> Bitsandbytes is also easily usable from within Accelerate, where you can quantize any model simply by passing a quantization config; e.g:

Reviewer: ```suggestion
Bitsandbytes is also easily usable from within Accelerate, where you can quantize any pytorch model simply by passing a quantization config; e.g:
```
